src/plugins/prebuilt-library/ai-wizard/constants/index.js

```diff
@@ -161,3 +136,49 @@ export const ENTITY_TO_NAME = {
 	'ORGANIZATION': __( 'Organization Name', 'kadence-blocks' ),
 };
 
+export const LOCATION_BUSINESS_ADDRESS = 'Business Address';
+export const LOCATION_SERVICE_AREA = 'Service Area';
+export const LOCATION_ONLINE_ONLY = 'Online Only';
+
+export const LOCATION_TYPES = [
+	{
+		icon: store,
+		text: __( 'Business Address', 'kadence-blocks' ),
+		value: LOCATION_BUSINESS_ADDRESS,
+		help: __( 'E.g.: 1234 Street #1, Chicago, IL 60076, USA', 'kadence-blocks' ),
+		placeholder: __( 'Street Adress, City, State, Zipcode, Country', 'kadence-blocks' ),
+	},
+	{
+		icon: mapMarker,
+		text: __( 'Service Area', 'kadence-blocks' ),
+		value: LOCATION_SERVICE_AREA,
+		help: __( 'E.g.: Chicago, USA', 'kadence-blocks' ),
+		placeholder: __( 'District, City, State, Zipcode, Country', 'kadence-blocks' ),
+	},
+	{
+		icon: desktop,
+		text: __( 'Online Only', 'kadence-blocks' ),
+		value: LOCATION_ONLINE_ONLY,
+		help: '',
+		placeholder: '',
+	},
+];
+
+export const MISSION_STATEMENT_GOAL = 300;
```

Comment:
300 seams pretty short? Is there any reason behind 300? or can we make it 400 or 500 ?

Reply:
Donaldo provided the following direction on the strength indicator: 

> Weak: <150 characters (~1 sentence long)
> Medium: 150-300 characters (~2 sentences long)
> Strong: >300 characters (~3+ sentences long)

I'm happy to ask about how we landed on 300 and ask about increasing the count in today's AI Scrum.